swlor2_2da/feat.2da

```diff
@@ -1880,3 +1880,14 @@
+1881   ForceDeath3                                      16860317   16860320      ife_forcedeath     ****             ****     ****     ****     ****     ****     ****     ****          ****          ****          0              0              1                  ****       ****    ****      ****        ****      ****         ****         ****         ****         ****         ****         ****         ****         ****       ****               ****        ****                ****                                             6                 ****          99         ****            ****       ****          0            0           
+1882   Tutaminis1                                       16860322   16860330      ife_tutaminis      ****             ****     ****     ****     ****     ****     ****     ****          ****          ****          0              0              1                  ****       ****    ****      ****        ****      ****         ****         1            ****         ****         ****         ****         ****         ****       ****               ****        ****                ****                                             6                 ****          99         ****            ****       ****          0            0           
+1883   Tutaminis2                                       16860323   16860331      ife_tutaminis      ****             ****     ****     ****     ****     ****     ****     ****          ****          ****          0              0              1                  ****       ****    ****      ****        ****      ****         ****         ****         ****         ****         ****         ****         ****         ****       ****               ****        ****                ****                                             6                 ****          99         ****            ****       ****          0            0             
+1884   Tutaminis3                                       16860324   16860332      ife_tutaminis      ****             ****     ****     ****     ****     ****     ****     ****          ****          ****          0              0              1                  ****       ****    ****      ****        ****      ****         ****         ****         ****         ****         ****         ****         ****         ****       ****               ****        ****                ****                                             6                 ****          99         ****            ****       ****          0            0           
```

Comment:
I don't see entries in the swlor2_tlk.tlk.json file for any of the Tutaminis abilities. Either you forgot to add them to the TLK or you forgot to run the tlk -> json converter script.